app/services/participants/sync_dqt_induction_start_date.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Participants
+  class SyncDqtInductionStartDate < BaseService
+    def initialize(dqt_induction_start_date, participant_profile)
+      @dqt_induction_start_date = dqt_induction_start_date
+      @participant_profile = participant_profile
+    end
+
+    def call
+      return false unless FeatureFlag.active?(:cohortless_dashboard)
+      return false if @dqt_induction_start_date.nil?
+      return false if @participant_profile.induction_start_date.present?
+
+      current_induction_record = @participant_profile.induction_records.latest
+      participant_cohort = current_induction_record.cohort
+      dqt_cohort = Cohort.containing_date(@dqt_induction_start_date)
+      return false if dqt_cohort.nil?
+
+      if dqt_cohort == participant_cohort
```

Comment:
Same as above. We should still call amend_cohort in this situation because even if the latest induction record is in the right cohort, some of the old induction records might still be in the wrong cohort and need to get it amended.

Reply:
if we do this and, as per [your comment above](https://github.com/DFE-Digital/early-careers-framework/pull/3265/files/44742161d9e37a4d0448ee836dfd61578b4df075#r1176695938), then this job will run all the time for each participant.
I think the ask was to do it just for the participants who did not have the induction start date set.
I'll double check whether this is still the case.

Reply:
Yes, exactly, because:
- we already have participants with wrong cohorts in their historical induction records and this is the tool to fix them, no matter their existing induction start date. 
- their existing induction date might not match the one in their DQT record and better to have the right one coming from DQT.
- we want to run this tool for ALL participants only one time to update their induction start date

Reply:
Once this one-off task is done, then your tool will only run for each new participant and for re-validations in support